app/src/main/res/layout/clock_user_bug.xml

```diff
@@ -1,39 +1,48 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent" android:layout_height="match_parent">
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_alignParentTop="true"
+    android:layout_alignParentEnd="true">
 
-    <TextClock
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:id="@+id/clock"
-        android:layout_alignParentTop="true"
-        android:layout_centerHorizontal="true"
-        android:format12Hour="h:mm"
-        android:format24Hour="k:mm"
-        android:textSize="40sp"
-        android:fontFamily="sans-serif-light" />
-
-    <LinearLayout
-        android:orientation="horizontal"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_below="@+id/clock"
-        android:layout_centerHorizontal="true"
-        android:gravity="center"
-        android:divider="@drawable/blank10x10"
-        android:showDividers="middle">
-
-        <ImageView
-            android:layout_width="15sp"
-            android:layout_height="15sp"
-            android:id="@+id/userImage"
-            android:src="@drawable/ic_user" />
+        <TextClock
+            android:id="@+id/clock"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_alignParentTop="true"
+            android:layout_centerHorizontal="true"
+            android:fontFamily="sans-serif-light"
+            android:format12Hour="h:mm"
+            android:format24Hour="k:mm"
+            android:shadowColor="@color/black"
+            android:shadowRadius="4"
+            android:textColor="@color/white"
+            android:textSize="40sp"
+            tools:text="13:37" />
 
-        <TextView
+        <LinearLayout
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:text="User Name"
-            android:id="@+id/userName"
-            android:fontFamily="sans-serif-light" />
-    </LinearLayout>
+            android:layout_below="@+id/clock"
+            android:layout_centerHorizontal="true"
+            android:gravity="center"
+            android:orientation="horizontal">
+
+            <ImageView
+                android:id="@+id/userImage"
+                android:layout_width="20dp"
+                android:layout_height="20dp"
+                android:layout_marginEnd="10dp"
+                android:src="@drawable/ic_user" />
+
+            <TextView
+                android:id="@+id/userName"
```

Comment:
This might be nitpicky but I prefer `username` instead of `userName`. Although just `name` might work too

Reply:
It is nitpicky, but I'll allow it. :sweat_smile: